build.gradle

```diff
@@ -250,7 +253,12 @@ sonarqube {
         property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/**," +
                 "src/main/java/uk/gov/hmcts/reform/judicialapi/JudicialApplication.java,"+
                 "src/main/java/uk/gov/hmcts/reform/judicialapi/domain/**," +
-                "src/main/java/uk/gov/hmcts/reform/judicialapi/controller/request/**"
+                "src/main/java/uk/gov/hmcts/reform/judicialapi/controller/request/**," +
+                "src/main/java/uk/gov/hmcts/reform/judicialapi/controller/TestingSupportController.java," +
+                "src/main/java/uk/gov/hmcts/reform/judicialapi/controller/feign/IdamUserFeignClient.java," +
+                "src/main/java/uk/gov/hmcts/reform/judicialapi/controller/response/**," +
+                "src/main/java/uk/gov/hmcts/reform/judicialapi/service/impl/IdamUserProfileServiceImpl.java," +
```

Comment:
Any reason why a Service Impl class is being excluded?

Reply:
@AdilOoze : As it is test endpoint, we do not want this to be executed in the hight environments.